src/main/java/de/pi3g/pi/oled/RotatableOLEDDisplay.java

```diff
@@ -0,0 +1,101 @@
+package de.pi3g.pi.oled;
+
+import com.pi4j.io.i2c.I2CFactory;
+
+import java.io.IOException;
+
+public class RotatableOLEDDisplay extends OLEDDisplay {
+
+    private byte ROTATION = 0;
```

Comment:
Why not use an enum here? Also according to the normal Java convention field names should not be upper case unless they are `static` and `final`.

Also maybe a constructor where this can be set directly would be nice.